.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+language: android
+android:
+  components:
+    - platform-tools
+    - tools
+    - build-tools-27.0.3
+    - android-27
+before_install:
+ - yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-27"
+before_script:
+ - chmod +x gradlew
+script:
+ - ./gradlew assembleDebug
```

Comment:
Great, one issue though
change `assembleDebug ` to `build `

Reply:
Ok I saw that on other projects as a comment! I'll change that now! thank you!!! @m-murad